README.md

```diff
@@ -372,37 +370,54 @@ For updating open swaps, I was successfully able to update 5 positions and recre
 
----
-## Frontend Agnostic
-By using the beacon tokens, it is trivial for any frontend to integrate with Cardano-Swaps. For example, the new Lace wallet by IOHK can integrate the DEX by simply adding support for querying the beacon tokens. They can also add their own user friendly way to create and use swaps. The only requirement is that all frontends agree to use the same beacon token standard. There is no need for risky extensions or dedicated frontends in order to use this DEX.
 
----
-## Conclusion
-This DEX protocol has all of the desired properties of a DEX:
+## FAQ
+
+#### If all users share a spending script, how are their assets protected?
+The spending script gets the staking credential from the address of the UTxO being spent at run-time. When an owner related action is being performed (closing or updating positions), the spending script requires that the staking credential "signals approval" of the action:
 
-1. Users maintain delegation control at all times.
-2. Composable atomic swaps allow creating arbitrarily complex swap transactions.
-3. Naturally concurrent and gets more concurrent the more available swaps there are.
-4. There is no impermanent loss since users can declare their desired minimum price.
-5. No secondary token is needed to interact with the DEX; only ADA is needed for the fees and deposits.
-6. Upgrades can happen democratically.
-7. Any frontend can easily add Cardano-Swap support.
+- If the staking credential is a pubkey, then the staking pubkey must sign the transaction.
+- If the staking credential is a script, then the script must be executed in the same transaction.
+- the staking credential effectively *becomes* the "owner" for all actions except for the actual swap execution, in which case the spending credential is used directly.
 
-Thanks to using beacon tokens, decentralization is no longer limited by the design of DEXs. Instead, the limiting factor is now the off-chain querying. However, innovations in this space are still in the early days. The Koios api is an example of a more decentralized off-chain platform. As the technology improves, the decentralization of this DEX will only improve.
+ :note: It is possible to execute a staking script even if 0 ADA is withdrawn from a reward address. The only requirement to use staking scripts like this is that the associated stake address must be registered and delegated. Stake addresses can be utilized this way as soon as the registration+delegation transaction is added to the chain. There is no epoch waiting-period.
```

Comment:
The :note: aren't being converted. I think it needs to either be memo or notebook ([emoji list](https://github.com/markdown-templates/markdown-emojis)).